### ISSUE TITLE
HeadFull mode is not working

### ISSUE THREAD

**Issue by luciferankon** [getgauge/taiko#594]
### Expected behavior
It should goto google.com and emit success event

### Actual behavior
It's getting hung up and timedout after 30000 ms

### Steps to reproduce
1. create a file  `foo.js` with this content
 ```const { openBrowser, goto, closeBrowser } = require('taiko');
(async () => {
    try {
        await openBrowser({headless:false});
        await goto('google.com');
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
2. run npm install and npm install -g .
3. run ./bin/taiko foo.js

### Versions
taiko version - commit hash c487ea4ef01f72e4e6b759d2328b61c9cc04b479
node version - v12.3.1

**Comment by nehashri** [comment]
Fix available in c39800b

**Comment by Debashis9012** [comment]
This issue has been validated and found fixed with the commit version ``` c39800b```
